### PR TITLE
#257 fix grammar issues

### PR DESCRIPTION
**Pull request by SinyukKirill**

### CODE REVIEW

**Comment by josdem** [comment]
Thanks for improving Vetlog. Every enhancement counts.  Adding @sindhunaydu and @marki2121 just in case they want to take a look too.

**Review by josdem — APPROVED** [review]

File: src/main/resources/i18n/messages.properties [hunk 1]
```diff
@@ -1,14 +1,14 @@
 app.title=Vetlog
 app.mailto=mailto:vetlog@josdem.io
 app.contact.us=Contact Us: vetlog@josdem.io
-app.description=Only in Mexico half a million dogs are euthanized every single year. Help us to decrease that number adopting instead!
+app.description=Only in Mexico are half a million dogs euthanized every single year. Help us to decrease that number by adopting instead!
```
Comment (marki2121) [inline src/main/resources/i18n/messages.properties:4]: app.description=Only in Mexico half a million dogs are euthanized every single year. Help us to decrease that number by adopting instead!


This is correct

Comment (SinyukKirill) [reply src/main/resources/i18n/messages.properties:4]: yeah. you're right. thanks

**Review by josdem — APPROVED** [review]
Thanks for doing suggested changes @SinyukKirill

**Comment by josdem** [comment]
@all-contributors please add @kirillsinyuk for translation

**Pull request merged**